components/raftstore/src/store/snap.rs

```diff
@@ -597,7 +597,9 @@ impl Snapshot {
             }
 
             if !plain_file_used(cf_file.cf) {
-                // Reset global seq number.
+                // TODO: remove reset_global_seq() if possible. TiKV does not write global_seqno
+                // into to-be-ingested SST files by default, this is only kept for backward
```

Comment:
`reset_global_seq` is to make sure the ingest can be retryable. now, it doesn't write global seqno anymore, so there is no need to reset global seq.

Reply:
This is for backward compatibility. See discussion above.